### PR TITLE
Add fake

### PR DESCRIPTION
**Pull request by khru**
## 📚 Description

Added `fake` function with several implementations.

## 🔖 Changes

This function covers:
- Fakes
- Mocks
- Spies
- Stubs

## ✅ To-do list

- [X] Make sure that all the pipeline passes
- [X] Make sure to update the `CHANGELOG.md` to reflect the new feature or fix

### CODE REVIEW

File: tests/unit/fake_test.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  fake ps 'echo ${FAKE_PARAMS[@]} >/tmp/fake_params'
+
+  code || true
+
+  assertEquals\
+    "$(Console::printFailedTest "Unsuccessful parameters of fake function" "ax" "but got" "a")"\
+    "$(assertEquals ax "$(head -n1 /tmp/fake_params)")"
+
+  unset code
+  unset _ps
```
Comment (Chemaclass) [inline tests/unit/fake_test.sh]: _ps ?

Comment (khru) [reply tests/unit/fake_test.sh]: It is a function that will mock the ps function

**Review by Tito-Kati — APPROVED** [review]
Awesome! ❤️❤️❤️

**Review by Chemaclass — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,2 @@
+PARALLEL_RUN=
```
Comment (Chemaclass) [inline .env.example:1]: Didn't we remove this feature?

Comment (khru) [reply .env.example:1]: image

I would say nop, but I can erase it

**Pull request merged**